docker/ftest-docker-junit-rule/src/test/java/org/arquillian/cube/docker/junit/rule/PostgreSqlIT.java

```diff
@@ -0,0 +1,99 @@
+package org.arquillian.cube.docker.junit.rule;
+
+import org.arquillian.cube.docker.impl.await.StaticAwaitStrategy;
+import org.arquillian.cube.docker.impl.client.config.Await;
+import org.arquillian.cube.docker.impl.requirement.RequiresDocker;
+import org.arquillian.cube.requirement.RequirementRule;
+import org.awaitility.Awaitility;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.attribute.PosixFilePermissions;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Verify that {@link ContainerDslRule} can be used to build a valid {@code PosatgreSql} Docker container.
+ * <p>
+ *     A {@link ContainerDslRule} is used to define a PostgreSql Docker container.
+ *     A local directory is created on the host to hold the PostgreSql data and to be mounted as a volume on the
+ *     Docker container.
+ *     The data directory is created statically, so that it can be used by the {@link ContainerDslRule} definition later
+ *     in the test lifecycle.
+ *     The {@code Z} bind mode option is passed so that it can be used by SELinux enabled systems, see the related
+ *     <a href="https://docs.docker.com/engine/storage/bind-mounts/#configure-the-selinux-label">Docker documentation</a>,
+ *     otherwise the container would fail to start with the following error:<br>
+ *     {@code mkdir: cannot create directory '/var/lib/pgsql/data/userdata': Permission denied}
+ * </p>
+ */
+@Category({ RequiresDocker.class})
```

Comment:
Shall we add `@RunWith(ArquillianConditionalRunner.class)` to make the `RequiresDocker` work as expected or am I missing something ?

Reply:
It is not always needed, as in this case, i.e. when using `@ClassRule` with `ContainerDslRule', see https://arquillian.org/arquillian-cube/#_junit_rule

Reply:
Ah, yes. that is the thing I am missing :) thank you .